hail_search/test_search.py

```diff
@@ -117,6 +117,36 @@ async def test_multi_project_search(self):
             sample_data=MULTI_PROJECT_SAMPLE_DATA,
         )
 
+    async def test_frequency_filter(self):
+        await self._assert_expected_search(
+            [VARIANT1, VARIANT4], frequencies={'seqr': {'af': 0.2}}, omit_sample_type='SV_WES',
```

Comment:
It is better to have a test for a population that has no frequencies or filters on a frequency other than 'af', 'ac', and 'hh'. It is the corner case that the `pop_filters` above can be empty.

Reply:
every other test in the file tests what happens when theres no frequency filter

Reply:
oh you mean has values that are all None. Yes, I can add a test